packages/compiler-sfc/src/cssVars.ts

```diff
@@ -13,7 +13,7 @@ import hash from 'hash-sum'
 
 export const CSS_VARS_HELPER = `useCssVars`
 export const cssVarRE =
-  /\bv-bind\s*\(\s*(?:'([^']+)'|"([^"]+)"|([^'"][^;]*))\s*\)/g
+  /\bv-bind\s*\(\s*(?:'([^']+)'|"([^"]+)"|([^'"]|[^;]*))\s*\)/g
```

Comment:
Thank you you for working on this fix.
However, I think the following CSS probably still can't be parsed well.

```css
width: calc(v-bind(foo) - 3px);
```

Reply:
thanks for your review.
I improved the code.